euRuKo proposal - 30 min

```diff
@@ -0,0 +1,31 @@
+euRuKo proposal - 30 min
+
+How this topic came about?
+I attended the Ruby on Ice conference last week which inspired me to create this talk.  It was obvious there was a desire to connect with other developers, have meaningful conversations and keep in touch post conference. However, there seemed to be a lack of know-how in doing this.  I took this personally (as a junior, I didn’t have enough technical knowledge or provide enough value to others) however upon speaking to 3 other senior developers, they said they also would’ve liked to keep in touch but didn’t. So I want to inspire a change in the tech community’s mindset and share some practical networking techniques to maximise the takeaway from the conference.
+
+My background
+* Only Junior/lady/Asian developer (yes to ticking all diversity boxes) at FromatoB  in Berlin
+* Been active part of the Ruby on Rails Community, Rails girls, Rubycorns in 3 different countries - Singapore, Malaysia and Germany
+* 2 years as Communications skills consultant for Singapore Government NTUC Largest people Union and Changi Airport.
+
+What my topic is all about?
+Connecting with conviction at Conferences. Geek style
+Pre conference preparation
+* What technical problems is your company facing? Be able to explain your architecture in 2 minutes
+* Check the social media (twitter, GitHub) of attendees and target specific people you want to speak to
+At the Conference - How to  rock at the ‘hallway track’
+*  Circle between groups, work the room, pretend you are the host
+* Say hi, what’s the worse that could happen?
+*  How to cut the awkward small talk and get straight to the point - ask specific questions eg. ‘Anyone struggling with kubernetes config? I see too many people ask about the company/product, do developers even care? It’s more likely you’ll share technologies than product/industry.
+* Have a clear idea what steam of conversation you want to take - Possible steams of conversation - Front end, backend, mobile? What kind of database? What kind of APIs do you provide? JSON? GraphQL
+* Add some comedy in your convo - How’s everyone’s SCRUM going! Let’s complain together
+Post conference - making long term connections
+*  Why is this important - share knowledge, recruiting
+* Keep in mind what social media suits you best (LinkedIn, twitter, telegram) - speed and smoothness is everything
+* Clarify what you can offer to the relationship, what they can provide you
+* Send ’It was great to connect with you’ - tips on things you could say
+
+Why this is important for EuRuko and the Ruby community?
```

Comment:
my two cents: 
maybe you can add more about why this is interesting for developers. what is the current problem and what are the benefits.